ui/app/rif/components/item-with-actions/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { faPen, faTimes } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+/**
+ *
+ * This component is a generic item with a edit/delete icon
+ * By default the icons are disabled, and must be explicitly rendered with the enable prop
+ * By default the edit click will render the children that are passed to it
+ */
+const ItemWithActions = (props) => {
```

Comment:
Same here, please use classes and proptypes inside the class as static member, we are using that as standar for this project.

Reply:
Of course, but I didn't know since it was not written anywhere.
Perhaps we should open an issue to include these things in the COTRNIBUGINT or the README